contracts/azorius/LinearERC20Voting.sol

```diff
@@ -151,16 +140,29 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent, BaseVotingBasisPe
             uint256 yesVotes,
             uint256 abstainVotes,
             uint32 startBlock,
-            uint32 endBlock
+            uint32 endBlock,
+            uint256 votingSupply
         )
     {
         noVotes = proposalVotes[_proposalId].noVotes;
         yesVotes = proposalVotes[_proposalId].yesVotes;
         abstainVotes = proposalVotes[_proposalId].abstainVotes;
         startBlock = proposalVotes[_proposalId].votingStartBlock;
         endBlock = proposalVotes[_proposalId].votingEndBlock;
+        votingSupply = getProposalVotingSupply(_proposalId);
     }
 
+    /** @inheritdoc BaseStrategy*/
+    function initializeProposal(bytes memory _data) public virtual override onlyAzorius {
```

Comment:
this function was moved lower in order to sit within the `public` functions, rather than `external`

the only code change was changing it to `public`